### PR TITLE
Added altitude to telemetry plugin

### PR DESCRIPTION
**Pull request by CY-1992**
This pull request would close [#1903](https://github.com/mavlink/MAVSDK/issues/1903).

### CODE REVIEW

File: protos/telemetry/telemetry.proto [hunk 1]
```diff
@@ -295,6 +301,11 @@ message HeadingResponse {
     Heading heading_deg = 1; // The next heading (yaw) in degrees
 }
 
+message SubscribeAltitudeRequest{}
+message AltitudeResponse{
```
Comment (JonasVautherin) [inline protos/telemetry/telemetry.proto:305]:
```suggestion
message SubscribeAltitudeRequest {}
message AltitudeResponse {
```

File: protos/telemetry/telemetry.proto [hunk 2]
```diff
@@ -803,6 +821,16 @@ message GpsGlobalOrigin {
     float altitude_m = 3 [(mavsdk.options.default_value)="NaN"]; // Altitude AMSL (above mean sea level) in metres
 }
 
+// Altitude message type
+// For more info see: https://mavlink.io/en/messages/common.html#ALTITUDE
```
Comment (JonasVautherin) [inline protos/telemetry/telemetry.proto:825]: We try not to leak MAVLink (too much) in the MAVSDK interface :innocent:.

```suggestion
```

**Review by JonasVautherin — APPROVED** [review]

**Review by JonasVautherin — CHANGES_REQUESTED** [review]
Whoops I meant "Request changes" for the two small comments I had :innocent:

**Comment by CY-1992** [comment]
Done the requested changes. Also updated proto module in MAVSDK. :smiley:

**Review by JonasVautherin — APPROVED** [review]

**Pull request merged**